packages/components/src/tabs/test/index.tsx

```diff
@@ -191,6 +195,8 @@ describe( 'Tabs', () => {
 		it( 'should focus on the related TabPanel when pressing the Tab key', async () => {
 			await render( <UncontrolledTabs tabs={ TABS } /> );
 
+			expect( await getSelectedTab() ).toHaveTextContent( 'Alpha' );
```

Comment:
All of these extra, initial assertions were added to increase test stability — basically, we are waiting for ariakit to initialise and we do that by waiting for the correct tab to be selected.